theforeman.org/pipelines/release/source/smart-proxy.groovy

```diff
@@ -26,7 +26,7 @@ pipeline {
                     }
                     stage("Test Ruby") {
                         steps {
-                            run_test(ruby)
+                            run_test(env.ruby)
```

Comment:
```suggestion
                            run_test(ruby: env.ruby)
```

Testing this change live -- https://ci.theforeman.org/blue/organizations/jenkins/smart-proxy-develop-source-release/detail/smart-proxy-develop-source-release/178/pipeline

Reply:
@ehelms Updated this change, but seems like I need to rebase it also.

> Testing this change live -- https://ci.theforeman.org/blue/organizations/jenkins/smart-proxy-develop-source-release/detail/smart-proxy-develop-source-release/178/pipeline